beacon_chain/utils/ssz/src/tree_hash.rs

```diff
@@ -0,0 +1,157 @@
+const CHUNKSIZE: usize = 128;
+const HASHSIZE: usize = 32;
+
+pub trait TreeHash {
+    // Note: it would be nice to have a default trait implementation here
```

Comment:
I didn't follow the error you're getting here, but you might be looking for the `Sized` trait. If you like, you can share the code that's giving you the error and I can figure it out. [Here](https://stackoverflow.com/questions/30938499/why-is-the-sized-bound-necessary-in-this-trait) is a fairly good discussion about `Sized`.

Reply:
Yes, the trait needed to be Sized and Encoded for the default method to compile.  However there are types implemented for this trait that are not Sized nor Encoded.  That seems preclude adding this default method - unless I'm missing something.